src/topology/algebra/uniform_ring.lean

```diff
@@ -226,3 +226,40 @@ begin
 end
 
 end uniform_space
+
+section uniform_extension
+
+variables {α : Type*} [uniform_space α] [semiring α]
+variables {β : Type*} [uniform_space β] [semiring β] [topological_semiring β]
+variables {γ : Type*} [uniform_space γ] [semiring γ] [topological_semiring γ]
+variables [t2_space γ] [complete_space γ]
+
+/-- The dense_inducing extension as a ring homomorphism. -/
+noncomputable def dense_inducing.extend_ring_hom {i : α →+* β} {f : α →+* γ}
```

Comment:
I don't think this should be in the `dense_inducing` namespace. Because it doesn't take an argument of that type.

Reply:
Thanks for your comments. 

The name comes from the fact that this is just the proof that `dense_inducing.extend_eq` is a ring homomorphism in this case. Also, it takes as arguments an `uniform_inducing` and a `dense_range` which gives basically a uniform version of `dense_inducing`. Anyway, I see your point, I could rename it to `uniform_inducing.extend_ring_hom_of_dense` for example.

Reply:
Hmm, if `dense_inducing` is already used elsewhere, then I guess it's fine. It makes for a nice and short name.